spec/models/filters/criteria/filter_for_dv_status_spec.rb

```diff
@@ -0,0 +1,114 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require_relative 'shared_filter_criteria_context'
+
+RSpec.describe Filters::Criteria::FilterForDvStatus do
+  include_context 'filter criteria setup'
+
+  let(:dv_status) { [1] } # DV survivor status
+  let(:filter) do
+    ::Filters::FilterBase.new(
+      user_id: user.id,
+      dv_status: dv_status,
+      start: start_date,
+      end: end_date,
+    )
+  end
+  let(:criteria) { described_class.new(input: filter, config: config) }
+
+  # Create enrollments with different DV statuses
+  let!(:enrollments) do
+    [
+      # Enrollment with DV survivor status (1)
+      create_enrollment_for_client(
+        create(:hud_client),
+      ),
+      # Enrollment without DV survivor status (0)
+      create_enrollment_for_client(
+        create(:hud_client),
+      ),
+      # Enrollment with different DV status (2)
```

Comment:
`DomesticViolenceSurvivor` uses list 1.8, which includes:
```
0 => yes
1 => no
8 => Client Doesn't know
9 => Client prefers not to answer
99 => Data not collected
```
For the purposes of the tests `2` is probably fine, but for consistency we should probably use valid options unless we're testing for bad data.